registry/handlers/app.go

```diff
@@ -232,6 +227,83 @@ func (app *App) register(routeName string, dispatch dispatchFunc) {
 	app.router.GetRoute(routeName).Handler(app.dispatcher(dispatch))
 }
 
+func (app *App) configureDriver(configuration *configuration.Configuration) {
+	for {
```

Comment:
@aaronlehmann It's just a way of keeping the code clean while re-writing the configuration. The issue is that an instance of `driver` is needed to make the decision.